amundsen_application/static/css/_variables-default.scss

```diff
@@ -103,6 +103,8 @@ $list-group-border-radius: 0 !default;
 // Labels
 $label-primary-bg: $brand-color-3 !default;
 
+//Priority
+$priority-text-blocker: $white; 
```

Comment:
nit: Can you also define some `$priority-bg-color`, set it to `$rose80`, and reference that in the styles. Always nicer to have things defined in the same place.

Reply:
oh neat, i didnt know i could use a hex value for rgba!

Reply:
ty!!

Reply:
Ah my comment was just about defining the use of $rose80 here. But if you want to use hex values for rgba, what would work is appending some more values according to the list defined here:
https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

In those examples they added the alpha codes to the beginning of the regular hex value, but when I tried this out what worked was adding it to the end. If for some reason you run into problems though, leaving the rgba is fine.